src/assets/scripts/client/info/AirportInfoController.js

```diff
@@ -0,0 +1,179 @@
+import { leftPad } from '../utilities/generalUtilities';
+import { radiansToDegrees } from '../utilities/unitConverters';
+import { INVALID_NUMBER } from '../constants/globalConstants';
+
+/**
+ * Gets information about the current airport, specifically
+ * the airport's elevation, wind speed and direction, and altimeter.
+ *
+ * @class SimAirportInfoController
```

Comment:
All the docblocks say this, but the class is called AirportInfoController

Reply:
done